homeassistant/components/media_player/kodi.py

```diff
@@ -109,6 +115,8 @@
 @asyncio.coroutine
 def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     """Set up the Kodi platform."""
+    if DATA_KODI not in hass.data:
+        hass.data[DATA_KODI] = []
```

Comment:
Let's instead make this a dict with the entity_id as the key, so we don't have to search it later.